src/bin/pr-metadata-validator.rs

```diff
@@ -234,9 +251,98 @@ async fn validate_pr(
         return Ok(ValidationResult::BodyTemplateNotFilledOut);
     }
 
+    let pr_assignment_descriptor_id =
+        get_descriptor_id_for_pr(&matched.sprints, pr_number).expect("This PR does not exist");
+    // This should never error, as a PR by this point in code must have been matched
+    // with an assignment, and PR assignments must have an associated issue descriptor
+
+    match check_pr_file_changes(
+        octocrab,
+        github_org_name,
+        module_name,
+        pr_number,
+        pr_assignment_descriptor_id,
+    )
+    .await
+    {
+        Ok(Some(problem)) => return Ok(problem),
+        Ok(None) => (),
+        Err(e) => {
+            let _ = anyhow!(e);
+        }
+    }
+
     Ok(ValidationResult::Ok)
```

Comment:
Do you intentionally want to swallow the error if there is one? If so, I'd suggest a comment, but if not:

```suggestion
        Ok(Some(problem)) => Ok(problem),
        Ok(None) => Ok(ValidationResult::Ok),
        Err(err) => Err(err),
    }
```